zyte_spider_templates/spiders/ecommerce.py

```diff
@@ -13,15 +14,29 @@
 from zyte_spider_templates.spiders.base import BaseSpider, BaseSpiderParams
 
 
+@document_enum
 class EcommerceCrawlStrategy(str, Enum):
     full: str = "full"
+    """Follow most links within the domain of URL in an attempt to discover and
+    extract as many products as possible."""
```

Comment:
Tried to use `Foo.bar.__doc__` in `enumMeta` to avoid duplication, but the `__doc__` is not generated at regular run time (there’s some “interactive” condition in the decorator implementation), and this way we can potentially provide more details in the reference docs than those we want in enumMeta; maybe we could even make enumMeta provide a link to the upstream reference for details where needed.